Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -265,6 +265,17 @@
     <string name="overclock_title">Emulated CPU Clock Speed</string>
     <string name="overclock_title_description">Adjusts the emulated CPU\'s clock rate if \"Override Emulated CPU Clock Speed\" is enabled.</string>
 
+    <!-- Log Configuration -->
+    <string name="log_configuration_submenu">Log Configuration</string>
```

Comment:
This should just say "Log", since it's shown on a screen that's entirely about configuration.